lua/caramba/core/commands.lua

```
@@ -88,4 +88,42 @@ function M.debug()
+      desc = opts.desc,
+      nargs = opts.nargs,
+      range = opts.range,
+      complete = opts.complete,
+    }
+  }
+end
+
+-- Setup all registered commands
+M.setup = function()
+  for name, cmd in pairs(M.commands) do
+    vim.api.nvim_create_user_command(name, cmd.func, cmd.opts)
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)